src/main/java/com/github/edouardswiac/zerotier/ZTService.java

```diff
@@ -25,4 +30,17 @@
   ZTNetworkMember getNetworkMember(String networkId, String address);
   void updateNetworkMember(ZTNetworkMember networkMember);
   void deleteNetworkMember(String networkId, String address);
+
+  List<ZTNet> getNets();
```

Comment:
I prefer explicit to implicit, and even though it's understood that ZTNet stands for ZTNetwork, the first committers to an open source project will set the bar for the quality of the code; I would say let's go with ZTNetwork :+1: